static/src/stylesheets/layout/new-header/_header-cta-item.scss

```diff
@@ -28,6 +28,13 @@
     @include mq(tablet) {
         font-size: 17px;
     }
+
+    .new-header--slim & {
```

Comment:
Not sure I like having all the styles for modification as a "parent class". Do you think I should instead put modifier classes on each thing I change to avoid nesting? It will make the template more complicated, but the css would be much nicer I think.

@gustavpursche